src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -946,6 +945,26 @@ ExecuteUtilityTaskListExtended(List *utilityTaskList, int poolSize,
 }
 
 
+/*
+ * ExecuteTaskList is a proxy to ExecuteTaskListExtended
+ * with defaults for some of the arguments.
+ */
+uint64
+ExecuteTaskList(RowModifyLevel modLevel, List *taskList)
+{
+	bool localExecutionSupported = true;
+	ExecutionParams *executionParams = CreateBasicExecutionParams(
+		modLevel, taskList, MaxAdaptiveExecutorPoolSize, localExecutionSupported
+		);
+
+	bool excludeFromXact = false;
+	executionParams->xactProperties = DecideTransactionPropertiesForTaskList(
+		modLevel, taskList, excludeFromXact);
+
+	return ExecuteTaskListExtended(executionParams);
+}
+
+
 /*
  * ExecuteTaskListOutsideTransaction is a proxy to ExecuteTaskListExtended
```

Comment:
We can remove `ExecuteTaskListOutsideTransaction` and `TRANSACTION_BLOCKS_DISALLOWED`? I always kept an eye on these APIs so that no one else uses them :)

Reply:
I guess it is used on the enterprise still, `ExecuteClusterOnCommands` and `ExecuteCreateIndexCommands`